content/en/monitors/status/graphs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Status Graphs
+further_reading:
+- link: "monitors/configuration/?tab=thresholdalert"
+  tag: "Documentation"
+  text: "Monitor configuration"
+- link: "monitors/configuration/?tab=thresholdalert/#group-retention-time"
+  tag: "Documentation"
+  text: "Group retention time"
+- link: "monitors/configuration/?tab=thresholdalert/#set-alert-conditions"
+  tag: "Documentation"
+  text: "Set alert conditions"
+---
+
+<div class="alert alert-info">Status Graphs is part of the <a href="/monitors/status/status_page">provisional Monitor Status Page</a>. If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation.</div>
+
+## Overview
+
+The graphs on the monitor status page provide insights into the single monitor evaluations. These graphs help you to understand why your monitor might be in an ALERT state and where to focus your troubleshooting efforts.
+
+## Monitor metadata
+
+{{< img src="/monitors/status/graphs/graphs_metadata.png" alt="Monitor metadata section on the right side of the status page." style="width:100%;" >}}
+
+The right panel of the graph section of the monitor status page provides a high level overview of your monitor including
+
+|  | Description |
+| ---- | ---- |
+| Groups  | Count of groups per status (Alert, Warn, No Data, Ok)  |
+| Visualize as | Graph selector to toggle between the Evaluated Data, Source Data, and In Group graphs. |
+| Query | The raw monitor query. A link to the Metrics explorer is also available\*. |
+| Evaluation | The aggregation method applied to the Query with the evaluation window. |
+| Notification count | Count of notifications sent from this monitor. |
+
+[**UPDATE**]
+\* The link to the Metrics Explorer is available only for Metrics, Event, Logs, … monitors.
+
+## Filter the page by groups or status
+
+Depending on the query, the monitor might have multiple groups. To focus on a specific group, use the filter dropdowns to select the desired group.
+
+{{< img src="/monitors/status/graphs/graphs_filter_by_group_status.png" alt="Example monitor status page filtered by the Alert status template variable" style="width:100%;" >}}
+
+You can chose to scope the page on:
+
+Group status
+: Only groups that are currently in the selected state will be shown
+
+Muted state
+: Only groups that are muted, or not, will be shown.
+
+Group tags
+: Only groups that have the selected tag will be shown.
+
+## Evaluated data graph
```

Comment:
This paragraph looks good to me, but since it's a completely new graph with so many interactions I would like to document it a little bit more

For instance one interaction that we should docuement is the fact that when users click on one phase (colored area in the graph) the event in the event timeline is selected. -> users can easily navigate from the graph to the respective event